packages/base-controller/src/BaseControllerV2.ts

```diff
@@ -117,7 +125,7 @@ export class BaseController<
     this.metadata = metadata;
 
     this.messagingSystem.registerActionHandler(
-      `${name}:getState`,
+      String(`${name}:getState`),
```

Comment:
What's the reasoning behind this?

Reply:
Replacing with a pattern used elsewhere in the file for now. I'll revisit the `any` usage here. 

```suggestion
      `${name}:getState` as Namespaced<ControllerName, any>,
```

Reply:
All of the `Namespaced` with `any` usage here has to do with the awkward way the `Namespaced` type is defined. I'm looking into ways we could do this differently.

Reply:
Addressed: ce08095bd1de347aa43046bfb6024348356ac8e8 and ae291da004b86a2984448ca0d71def73ba0ac97f